ambry-tools/src/main/java/com.github.ambry.tools/admin/DumpData.java

```diff
@@ -55,103 +43,134 @@
+              .ofType(String.class);
 
       ArgumentAcceptingOptionSpec<String> typeOfOperationOpt = parser.accepts("typeOfOperation",
           "The type of operation to be performed - DumpLog or DumpIndex or DumpIndexesForReplica or " +
-              "DumpActiveBlobsFromIndex or DumpActiveBlobsForReplica or DumpNRandomActiveBlobsForReplica or " +
-              "DumpReplicatoken or CompareIndexToLog").withRequiredArg()
-          .describedAs("The type of Operation to be " + "performed").ofType(String.class).defaultsTo("log");
+              "or DumpNRandomActiveBlobsForReplica or DumpReplicatoken or CompareIndexToLog or "
+              + "CompareReplicaIndexesToLog or CompareLogToIndex")
+          .withRequiredArg()
+          .describedAs("The type of Operation to be " + "performed")
+          .ofType(String.class);
 
       ArgumentAcceptingOptionSpec<String> listOfBlobs =
-          parser.accepts("listOfBlobs", "List Of Blobs to look for during log/index dump").withRequiredArg()
-              .describedAs("List of blobs, comma separated").ofType(String.class);
+          parser.accepts("listOfBlobs", "List Of Blobs to look for while performing log or index dump operations")
+              .withRequiredArg()
+              .describedAs("List of blobs, comma separated")
+              .ofType(String.class);
 
       ArgumentAcceptingOptionSpec<String> startOffsetOpt =
-          parser.accepts("startOffset", "Log Offset to start dumping from").withRequiredArg().describedAs("startOffset")
+          parser.accepts("startOffset", "Log Offset to start dumping from log")
+              .withRequiredArg()
+              .describedAs("startOffset")
               .ofType(String.class);
 
       ArgumentAcceptingOptionSpec<String> endOffsetOpt =
-          parser.accepts("endOffset", "Log Offset to end dumping").withRequiredArg().describedAs("endOffset")
+          parser.accepts("endOffset", "Log Offset to end dumping in the log")
+              .withRequiredArg()
+              .describedAs("endOffset")
               .ofType(String.class);
 
-      ArgumentAcceptingOptionSpec<String> logFileToCompareOpt =
-          parser.accepts("logFileToDump", "Log file that needs to be dumped for Operation \"CompareIndexToLog\" ")
-              .withRequiredArg().describedAs("log_file_to_dump").ofType(String.class);
+      ArgumentAcceptingOptionSpec<String> logFileToCompareOpt = parser.accepts("logFileToDump",
+          "Log file that needs to be dumped for comparison operations like \"CompareIndexToLog\" "
+              + "\"CompareReplicaIndexesToLog\" and \"CompareLogToIndex\"")
+          .withRequiredArg()
+          .describedAs("log_file_to_dump")
+          .ofType(String.class);
 
       ArgumentAcceptingOptionSpec<String> outFileOpt =
-          parser.accepts("outFile", "Output file to redirect the output ").withRequiredArg().describedAs("outFile")
+          parser.accepts("outFile", "Output file to redirect the output to")
```

Comment:
I am not sure why we have this inconsistency. I am also using the same formatter